### PR TITLE
Updated Mojave build instructions for libxml2

### PR DESCRIPTION
**Pull request by cjohansson**
[#886](https://github.com/Wilfred/remacs/issues/886)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,12 +149,12 @@ more Emacs-y.
...
    MacOS:
    
-        brew install libxml2 gnutls texinfo
+        brew install gnutls texinfo
         
     To use the installed version of `makeinfo` instead of the built-in 
     (`/usr/bin/makeinfo`) one, you'll need to make sure `/usr/local/opt/texinfo/bin` 
     is before `/usr/bin` in `PATH`.
-    Mojave: `export PKG_CONFIG_PATH="/usr/local/opt/libxml2/lib/pkgconfig"`
+    Mojave: `export PKG_CONFIG_PATH="/Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg"`
```
Comment (shaleh) [inline README.md]: Does the user need to install this first? Or is only pointing at the `.pkg` sufficient.

Comment (cjohansson) [reply README.md]: No I think this should be available once Xcode Command Line tools are installed and they are a requirement for compiling stuff on macOS, you install them with `xcode-select --install`

**Comment by cjohansson** [comment]
I added instructions for installing the headers, it seems the export command doesn't work, it must have been something else in my environment that made it appear to work last time

**Comment by shaleh** [comment]
Thank you for your diligence @cjohansson.

**Pull request merged**